external/wycheproof/test.properties

```diff
@@ -3,5 +3,5 @@ script="wycheproof.sh"
 home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="master"
-bazel_version="1.2.1"
+bazel_version="4.2.2"
```

Comment:
I think this change should be part of #3216 to update the bazel version, which should not be part of this PR. This PR is a general fix of parsing and setting set information.